astro/src/content/docs/extend/examples/modeling-organizations.mdx

```diff
@@ -0,0 +1,260 @@
+
+The `grant` object, or some portion of it, can be stored in any number of locations for easy access. We could store it in a cookie (encrypted is preferred), a server-side session, or a database. The [Search Grants](/docs/v1/tech/apis/entity-management/grants#search-for-grants) is designed to be queried repeatedly and returned quickly, so depending on the scale of the application and where FusionAuth is deployed, we could also query this API for each request. This has the benefit of ensuring that the User's permissions are always the most current version.
+
+## Multiple Organizations
+
+The final component we'll cover in this guide is handling multiple organizations for a single User. AwesomeCRM works will many partners and often these partners work with many clients. This means that a User at a partner might be working with multiple organizations at the same time. To handle this scenario, we simply need to ensure we can handle multiple results from the [Retrieve Grants](/docs/v1/tech/apis/entity-management/grants#retrieve-grants) API.
+
+Here's an example of the response from the [Retrieve Grants](/docs/v1/tech/apis/entity-management/grants#retrieve-grants) API when a user has multiple organizations (many properties have been trimmed for brevity):
+
+```json
+{
+  "grants": [
+    {
+      "entity": {
+        "name": "Pied Piper",
+        ...
+      },
+      "permissions": [
+        "Admin"
+      ],
+      ...
+    },
+    {
+      "entity": {
+        "name": "Hooli",
+        ...
+      },
+      "permissions": [
+        "Sales"
+      ],
+      ...
+    },
+    {
+      "entity": {
+        "name": "Aviato",
+        ...
+      },
+      "permissions": [
+        "Billing",
+        "Reports"
+      ],
+      ...
+    }
+  ],
+  "total": 3
+}
+```
+
+This response indicates that the User belongs to 3 organizations and has various permissions to each. To manage this in our AwesomeCRM application, we can present an option to the User when they log in, to select the company they'd like to work on initially. This option screen might look like this:
+
+[IMAGE]
+
+## Managing Users
+
+Often applications provide the ability for users to manage other users on their account. This is accomplished in much the same way we have described above. To add a user to an organization, we can use the [Grant API](/docs/v1/tech/apis/entity-management/grants#grant-a-user-or-entity-permissions-to-an-entity). To remove a user from an organization, we can use the [Delete a Grant](/docs/v1/tech/apis/entity-management/grants#delete-a-grant) API. If we need to adjust the permissions a user has to an organization, we can use the same [Grant API](/docs/v1/tech/apis/entity-management/grants#grant-a-user-or-entity-permissions-to-an-entity) from above. The Grant API is an upsert, which means that if a grant already exists, it will update its attributes.
```

Comment:
```suggestion
Often applications provide the ability for users to manage other users on their account. This is accomplished in much the same way as described above. To add a user to an organization, you can use the [Grant API](/docs/v1/tech/apis/entity-management/grants#grant-a-user-or-entity-permissions-to-an-entity). To remove a user from an organization, you can use the [Delete a Grant](/docs/v1/tech/apis/entity-management/grants#delete-a-grant) API. If you need to adjust the permissions a user has to an organization, you can use the same [Grant API](/docs/v1/tech/apis/entity-management/grants#grant-a-user-or-entity-permissions-to-an-entity) from above. The Grant API is an upsert, which means that if a grant already exists, it will update its attributes.
```